test/end-to-end/helpers/index.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const Promise = require('bluebird');
+const querystring = require('querystring');
+const uuid = require('uuid');
+
+const helpers = {};
+
+helpers.getAuthClientParams = function() {
+  return {
+    // eslint-disable-next-line no-process-env
+    clientId: process.env.INTEGRATION_CLIENT_ID,
+    // eslint-disable-next-line no-process-env
+    clientSecret: process.env.INTEGRATION_CLIENT_SECRET,
+    redirectUri: 'https://example.com/auth',
+    development: true,
+  };
+};
+
+const getCodeFromUri = function(uri) {
+  // this helper functions assumes the structure of the uri
+  // is valid and contains the required code
+  const firstIndexOfParams = uri.indexOf('?') + 1;
+  const params = uri.substring(firstIndexOfParams);
+
+  return querystring.parse(params).code;
```

Comment:
we could use Node's built in URL parsers: https://nodejs.org/docs/latest-v6.x/api/url.html#url_class_urlsearchparams
that we don't have to make any assumptions